lib/orm.php

```diff
@@ -1951,13 +1968,13 @@ protected function run() {
 		global $wpdb;
 
 		$query   = $this->build_select();
-		$success = self::execute( $query, $this->values );
+		$success = self::execute( $query, $this->values, $this->id() );
```

Comment:
`$this->id()` results in the column name of the primary key instead of a unique identifier for the object, which isn't unique across different models. Given the nature of the ORM object, I don't think we can determine a unique resource identifier for a single (indexable) object (if that is what you were going for). We could consider grouping caches by `$this->table_name` instead. 
This would ofcourse mean that cache for _all_ queries for a model are purged when a model changes, but I don't think that's a problem with such short lived caches.

Reply:
Replaced `$this->id()` with `$this->table_name`. I agree, since the caches are relatively short-lived, it makes sense to use the table-name 👍